docs/for_developers.md

```diff
@@ -74,7 +74,7 @@ flowchart LR
 
     === "Linux"
         ```bash
-        CELERY_BROKER_URL=redis://localhost:6379/0 CELERY_RESULT_BACKEND=redis://localhost:6379/0 poetry run celery --app yaptide.celery.worker worker -P eventlet --loglevel=debug
+        PATH=$PATH:bin/ BACKEND_INTERNAL_URL=http://localhost:5000 CELERY_BROKER_URL=redis://localhost:6379/0 CELERY_RESULT_BACKEND=redis://localhost:6379/0 poetry run celery --app yaptide.celery.worker worker -P eventlet --loglevel=debug
```

Comment:
@jagodek adjust powershell below as well with PATH and BACKEND_INTERNAL_URL modifications

Reply:
![image](https://github.com/yaptide/yaptide/assets/7374733/d8de0c4d-cc39-4393-8320-4c2027f3e7f8)

probably some quotes are missing

Reply:
![image](https://github.com/yaptide/yaptide/assets/7374733/6c1e3174-fbac-4a46-ae49-5559a033d29e)